system/service_systemd.go

```
@@ -73,7 +77,9 @@ func (s *ServiceSystemd) Running() (bool, error) {
 		return false, nil
 	}
 	cmd := util.NewCommand("systemctl", "-q", "is-active", s.service)
-	cmd.Run()
+	if err := cmd.Run(); err != nil {
+		return false, nil
```

these should probably bubble up the error:

```
return false, err
```